src/snap/start.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# If there is file with plugin URLs, download those plugins to autoload directory.
+if [ -f /etc/snap/plugins.yaml ]
+  then
+    echo "-----------------------Downloading plugins-----------------------"
+    while read plugin_address; do
+      plugin=`echo $plugin_address | rev | cut -d / -f 1 | rev`
+      if [ "$plugin" == "" ]
+        then
+          continue
+      fi
+      echo "wget $plugin_address -O /opt/snap/autoload/$plugin"
```

Comment:
Maybe, let's try to make autoload dir configurable, as it can be set by user in snapteld.conf